zerver/management/commands/enqueue_file.py

```diff
@@ -12,6 +12,24 @@ def error(*args: Any) -> None:
     raise Exception("We cannot enqueue because settings.USING_RABBITMQ is False.")
 
 
+def enqueue_file(queue_name: str, f: IO[str]) -> None:
+    for line in f:
```

Comment:
Does this need to be `in f.readlines()`?

Reply:
No. A file is an iterator and `.readlines()` is an unnecessary intermediate list.